### PR TITLE
hi.js V5.7.0 update changes

### PR DESCRIPTION
**Pull request by TheDuce**

### CODE REVIEW

File: hi.js [hunk 1]
```diff
@@ -220,11 +220,11 @@ settings.interface.nametagStyle.all सब कुछ
 settings.interface.healthNum नैम-टैग HP संख्या
 settings.interface.healthColT नैम-टैग अपनी टीम के HP का रंग
 settings.interface.healthColE नैम-टैग दुश्मन के HP का रंग
-settings.interface.xpBarType XP Bar Type
+settings.interface.xpBarType XP बार का टाइप
 settings.interface.xpBarCol XP बार का रंग
 settings.interface.xpBarCol2 XP Bar Color 2
 settings.interface.xpBarOpac XP बार की पारदर्शकता
-settings.interface.adsObjOpac ADS Objective Opacity
+settings.interface.adsObjOpac ADS ऑब्जेक्टिव अपारदर्शिता
```
Comment (HistoriFy) [inline hi.js:227]: अपारदर्शिता too complicated, leave the term as it is

File: hi.js [hunk 2]
```diff
@@ -564,7 +564,7 @@ windows.controls.premium1 गाड मोड
 windows.controls.premium2 खुद को नष्ट करे
 windows.controls.premium3 स्माइट
 windows.controls.reset स्थान रिसेट करे (रेस & पार्कोर)
-windows.controls.resetLast Reset To Last Checkpoint (Parkour etc)
+windows.controls.resetLast रिसेट टू लास्ट चेकपॉइंट (Parkour etc)
```
Comment (HistoriFy) [inline hi.js:567]: no need to convert english statement in hindi. It should be complete, easy translation

File: hi.js [hunk 3]
```diff
@@ -858,21 +858,21 @@ stats.assists असिस्ट
 stats.nukes न्यूक
 stats.slimers Slimers
 stats.juggernauts Juggernauts
-stats.juggernauts.kills Juggernaut Kills
-stats.juggernauts.killed Juggernauts Killed
+stats.juggernauts.kills Juggernaut किल्स
```
Comment (HistoriFy) [inline hi.js:861]: again no need of mixed translations

File: hi.js [hunk 4]
```diff
@@ -858,21 +858,21 @@ stats.assists असिस्ट
...
+stats.juggernauts.kills Juggernaut किल्स 
+stats.juggernauts.killed Juggernauts किल्ड 
 stats.warmachines War Machines
 stats.melee चाकू किल
 stats.fistkills हाँथों द्वारा किये गए किल
 stats.thrownkills फेंके गए चाक़ू द्वारा किल
 stats.headshots हेड शाट
 stats.wallbangs वॉल बैंग
-stats.crouches Crouches
+stats.crouches क्रोचेस
```
Comment (HistoriFy) [inline hi.js:869]: should be क्राउचेस (crou-ches) not क्रोचेस (kro-ches)

File: hi.js [hunk 5]
```diff
@@ -858,21 +858,21 @@ stats.assists असिस्ट
...
 stats.sprays स्प्रे काउन्ट
 stats.kpd मददगार रिपोर्टस
-stats.shots Shots
-stats.hits Hits
-stats.misses Misses
-stats.legshots Legshots
+stats.shots शॉट्स 
+stats.hits हिट्स 
+stats.misses मिसेस 
+stats.legshots लेग्शॉट्स
```
Comment (HistoriFy) [inline hi.js:875]: लेग्शॉट्स will feel complicated. Keep it as it is . P.S. - लेगशाट्स is the easier translation here

File: hi.js [hunk 6]
```diff
@@ -1006,8 +1006,8 @@ server.message.kicked {0} को किक किया गया
 server.message.banned {0} को बैन किया गया
 server.message.votekicked {0} was vote kicked
 server.message.arrested {0} को K.P.D ने गिरफ्तार कर लिया
-event.easter.golden.spawn A Golden egg is near
-event.easter.golden.despawn A Golden egg has gone
+event.easter.golden.spawn A गोल्डन एग is near
+event.easter.golden.despawn A गोल्डन एग has gone
```
Comment (HistoriFy) [inline hi.js:1010]: same thing, no need of mixed statements

File: hi.js [hunk 7]
```diff
@@ -1006,8 +1006,8 @@ server.message.kicked {0} को किक किया गया
...
 event.easter.found.new {0} found {1}/{2} Eggs
-event.easter.found.old You already discovered this egg!
-event.easter.completed {0} completed the Egg hunt!
+event.easter.found.old You already discovered this एग!
```
Comment (HistoriFy) [inline hi.js:1012]: mixed statement again

**Review by HistoriFy — CHANGES_REQUESTED** [review]
Reviewed v.5.7.0 #474

File: hi.js (hunk 1 above)
Comment (TheDuce) [reply hi.js:227]: so opacity in hindi or just opacity,

Comment (TheDuce) [reply hi.js:227]: i took that word from other opacity translations

File: hi.js (hunk 6 above)
Comment (mty22) [reply hi.js:1010]: just leave this in English it's partially translated.

**Review by notanerdstudent — APPROVED** [review]

**Comment by mty22** [comment]
Treating this one as stalled. Please re-open the PR if you wish to review.

**Pull request closed without merging**